csharp/src/SeedLang.Shell/Repl.cs

```diff
@@ -23,20 +23,31 @@ namespace SeedLang.Shell {
   internal sealed class Repl {
     private class Visualizer : IVisualizer<AssignmentEvent>,
                                IVisualizer<BinaryEvent>,
+                               IVisualizer<CompareEvent>,
                                IVisualizer<EvalEvent> {
       // Current executed source code.
       public string Source { get; set; }
 
-      private readonly Dictionary<BinaryOperator, string> _operatorStrings =
-        new Dictionary<BinaryOperator, string>() {
-          {BinaryOperator.Add, "+"},
-          {BinaryOperator.Subtract, "-"},
-          {BinaryOperator.Multiply, "*"},
-          {BinaryOperator.Divide, "/"},
-          {BinaryOperator.FloorDivide, "//"},
-          {BinaryOperator.Power, "**"},
-          {BinaryOperator.Modulus, "%"},
-        };
+      private readonly Dictionary<BinaryOperator, string> _binaryOperatorStrings =
+          new Dictionary<BinaryOperator, string>() {
+            {BinaryOperator.Add, "+"},
+            {BinaryOperator.Subtract, "-"},
+            {BinaryOperator.Multiply, "*"},
+            {BinaryOperator.Divide, "/"},
+            {BinaryOperator.FloorDivide, "//"},
+            {BinaryOperator.Power, "**"},
+            {BinaryOperator.Modulus, "%"},
+          };
+
+      private readonly Dictionary<CompareOperator, string> _compareOperatorStrings =
+          new Dictionary<CompareOperator, string>() {
+            {CompareOperator.Less, "<"},
+            {CompareOperator.Great, ">"},
+            {CompareOperator.LessEqual, "<="},
+            {CompareOperator.GreatEqual, ">="},
+            {CompareOperator.EqualEqual, "=="},
```

Comment:
`EqEqual`? to be consistent with https://github.com/python/cpython/blob/main/Parser/token.c ?

Reply:
Done